cmd/bnscli/cmd_gov.go

```diff
@@ -361,3 +372,77 @@ Tally triggers the tally execution after the voting period had ended.
 	_, err := writeTx(output, govTx)
 	return err
 }
+
+func cmdTextResolution(input io.Reader, output io.Writer, args []string) error {
+	fl := flag.NewFlagSet("", flag.ExitOnError)
+	fl.Usage = func() {
+		fmt.Fprintln(flag.CommandLine.Output(), `
+Text resolution is  'as-proposal' command.
+		`)
+		fl.PrintDefaults()
+	}
+	var (
+		textFl = fl.String("text", "", "Human readable resolution text")
+	)
+	fl.Parse(args)
+	if len(*textFl) == 0 {
+		flagDie("the text must not be empty")
+	}
+	msg := &gov.CreateTextResolutionMsg{
+		Metadata:   &weave.Metadata{Schema: 1},
+		Resolution: *textFl,
+	}
+	// serialize and write to output
```

Comment:
This comment states what `Marshal` and `Write` does. It does not explain _why_. I am not sure if it has any value.

Reply:
comment :oncoming_police_car: